pkg/process/util/network_filter.go

```diff
@@ -44,25 +49,68 @@ func ParseConnectionFilters(filters map[string][]string) (blacklist []*Connectio
 
 		validFilter := true
 		for _, v := range ports {
-			if v == "*" {
+			// reset validFilter for each port
+			validFilter = true
+			v = strings.ToUpper(v)
+			// set tcp and udp to true to prevent exclusion by default
+			tcp, udp := true, true
```

Comment:
This function is quite long, can we break most of this loop into it's own function?

```
func parsePortAndProtocolFilter(string) (lowerPort int, upperPort int, ConnectionTypeFilter)
```